cms_form/models/cms_search_form.py

```diff
@@ -34,6 +34,12 @@ class CMSFormSearch(models.AbstractModel):
         # opt 2: function that give back `(fname, operator, value)``
         # 'foo': lambda field, value, search_values: ('foo', 'not like', value)
     }
+    # xpath expression to find the DOM elements representing results in a
+    # rendered search result template
+    _form_ajax_items_xpath =\
```

Comment:
further thinking...

We could add a new generic class to the wrapper of the content, such as `form_content` which would be generic (AFAIS we could wrap [this block](https://github.com/OCA/website-cms/blob/11.0/cms_form/templates/form.xml#L156-L164) into `div.form_content`) 
Then add a property:
```py
@property
def form_content_selector(self):
    return  '.'.join(form.form_css_klass.split()) + ' .form_content'
```
Then we put the selector into `data-form-content-selector` or similar attribute on the form elem so that we can easily reuse the same logic to replace the content of any kind of form on the fly (eg: edit form submit w/ errors).

I hope is clear :)